HelpSource/scdoc.js

```diff
@@ -412,14 +402,43 @@ function toc_search(ev) {
     }
 }
 
+
+function set_up_toc() {
+    var toc_container = $("<div>", {id: "toc-container"})
+        .insertBefore($("#menubar").children().first());
+
+    var toc_link = $("<a>", {
+        href: "#",
+        class: "menu-link",
+        text: "Table of contents \u25bc"
+    }).appendTo(toc_container);
+
+    $("#toc").appendTo(toc_container);
+
+    toc_items = document.getElementById("toc")
+        .getElementsByTagName("ul")[0].getElementsByTagName("li");
```

Comment:
@snappizz why not use `zepto` selector here instead of vanilla js? Can't you just do (like jquery):
```js
$('#toc ul').eq(0).find('li')
```
Same below with `$('#toc_search')`?

Reply:
@snappizz you use `zepto` selectors one line above so please, let's either use it or not, but not both